### PR TITLE
[Docs] how to get channels of a new backbone

### PR DESCRIPTION
**Pull request by jbwang1997**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add docs of how to get channels of a new backbone.
solve #7349

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by Czm369** [comment]
LGTM

**Comment by BIGWangYuDong** [comment]
should fix the conflict before merge into dev

File: docs/en/tutorials/how_to.md [hunk 1]
```diff
@@ -173,3 +173,32 @@ class UnfreezeBackboneEpochBasedHook(Hook):
                 for param in m.parameters():
                     param.requires_grad = True
 ```
+
+## Get the channels of a new backbone
+
+If you want to get the channels of a new backbone, you can build this backbone alone and input a pseudo image to get each stage output.
+
+Take `ResNet` for an example:
```
Comment (ZwwWayne) [inline docs/en/tutorials/how_to.md:181]: Take xxx as an example

File: docs/en/tutorials/how_to.md [hunk 2]
```diff
@@ -173,3 +173,32 @@ class UnfreezeBackboneEpochBasedHook(Hook):
...
+self = ResNet(depth=18)
+self.eval()
+inputs = torch.rand(1, 3, 32, 32)
+level_outputs = self.forward(inputs)
+for level_out in level_outputs:
+    print(tuple(level_out.shape))
+
+```
+
+The outputs of above script is:
```
Comment (ZwwWayne) [inline docs/en/tutorials/how_to.md:195]: output of the above script is as below:

File: docs/en/tutorials/how_to.md [hunk 3]
```diff
@@ -173,3 +173,32 @@ class UnfreezeBackboneEpochBasedHook(Hook):
...
+
+```python
+(1, 64, 8, 8)
+(1, 128, 4, 4)
+(1, 256, 2, 2)
+(1, 512, 1, 1)
+```
+
+Replacing the `ResNet(depth=18)` in this script with your backbone and its setting, and you can get the channels of the new backbone.
```
Comment (ZwwWayne) [inline docs/en/tutorials/how_to.md:204]: Users can get the channels of the new backbone by replacing the `ResNet(depth=18)` in this script with their customized backbone.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**